apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -72,18 +72,19 @@ data class Avkorting(
         bruker: BrukerTokenInfo,
         beregning: Beregning,
     ) = if (behandlingstype == BehandlingType.FØRSTEGANGSBEHANDLING) {
-        oppdaterMedInntektsgrunnlag(nyttGrunnlag, virkningstidspunkt, bruker).beregnAvkortingForstegangs(beregning)
+        oppdaterMedInntektsgrunnlag(nyttGrunnlag, virkningstidspunkt, true, bruker).beregnAvkortingForstegangs(beregning)
     } else {
-        oppdaterMedInntektsgrunnlag(nyttGrunnlag, virkningstidspunkt, bruker).beregnAvkortingRevurdering(beregning)
+        oppdaterMedInntektsgrunnlag(nyttGrunnlag, virkningstidspunkt, false, bruker).beregnAvkortingRevurdering(beregning)
```

Comment:
I stedet for et boolsk flagg kunne denne tatt inn f.eks. første virk i saken? Hvis den er null (evt samme som virk i denne behandlingen) så oppdaterer man årsoppgjøret med forventet innvilgede måneder. Det blir kanskje en lettere endring i ettertid å støtte flytting av virk for saken